src/frontend/src/lib/derived/network-tokens.derived.ts

```diff
@@ -32,6 +33,14 @@ export const enabledErc20NetworkTokens: Readable<Erc20Token[]> = derived(
 		$enabledNetworkTokens.filter(({ standard }) => standard === 'erc20') as Erc20Token[]
 );
 
+export const enabledIcNetworkTokens: Readable<IcToken[]> = derived(
```

Comment:
Can you add a comment on why we have this derived store? We have comments on other stores of this particular module

Reply:
done!